taas/mapping.py

```diff
@@ -0,0 +1,99 @@
+from abc import ABCMeta, abstractmethod
+
+
+class Mapping(object):
+    """
+        This class handles mappings between our raw input, and what we generate.
```

Comment:
https://github.com/vinitkumar/googlecl/blob/master/googlecl-pylint.rc + pylint might do for you?